### DISCUSSION TITLE
Life After POE?

### DISCUSSION THREAD

**Discussion by SomebodySysop** [gtxaspec/wz_mini_hacks#740] [Development]
I have around 6 Wyze V3 cameras that I deployed using the UCTronics POE adapter and wz mini hacks.  Most have run for several months.  I noticed that when I tried to revert back to Wyze firmware and use them with the Wyze Adapter and Cable they came with, most will not  work.  You can't factory refresh or flash firmware because they simply will not boot up.  

Of 5 I have tried so far, I was only able to re-flash the Wyze firmware on one using the Wyze cable and adapter.  Two I am only able to boot up using the POE adapter, and two will not boot under any circumstances.  Anyone else seen this?

Any suggestions other than try a different cable/adapter?

**Comment by virmaior** [comment]
very strange.
it's quite likely that at some point `/configs` was getting clobbered on the cameras rather than having used the POE adapter borking them. 

Can you get any logs for any of the cameras with problems?

**Reply by SomebodySysop** [reply]
_Can you get any logs for any of the cameras with problems?_

I can't boot either one of the cameras at all, no matter what power supply or cable I use.

**Comment by Bob-AOL** [comment]
What is the purpose of re-flashing the firmware..., or reverting to stock OEM operation at all?

Call me curious.

**Reply by SomebodySysop** [reply]
_What is the purpose of re-flashing the firmware..., or reverting to stock OEM operation at all?_

At this point, I can do neither.  I was just trying to salvage the cameras for future use.  But as it stands, nothing I've done works.  I have not tried opening them up, just trying to turn on using, as I've said before, a variety of cables/power supplies.

The one curious thing I did note with one camera -- when connecting using POE connection, the camera connector got extremely hot even though it never switched on.  Never noticed anything like that before.

So, just trying to get them to work before I revert them to paper weights.

**Reply by Bob-AOL** [reply]
They won't even respond to a reset using the initial startup and registration method holding in the button on the bottom?

I presume the wz_mini SD cards have been removed? Any logs would be on them.

**Reply by SomebodySysop** [reply]
I wiped the wz_mini SD cards clean when I was attempting to re-flash Wyze firmware.  Holding button down before and/or after plugging in -- neither process works.

**Reply by Bob-AOL** [reply]
Depending on the POE source(s) and connectorization/environmental conditions, the cameras may indeed have been damaged beyond any reasonable effort to repair them. I would at least remove the lenses for possible future use if deciding to toss them. They're a decent lens w/standard thread and can be used in many cameras.

**Comment by virmaior** [comment]
If you have a known working SD card that still has wz_mini on it, you can go step by step and use that to figure out how badly the other cameras are borked.
1. Test SD card in working camera. 
2. See if it runs wz_mini.
3. check log files on SD card to see that they were produced
4. pop into non working camera
5. check for new log files

if no new log files, then try adding something that logs to the beginning of the boot process on the SD card in `/etc/init.d/` which loads the files based on their numerical order.

specifically, you're going to want to do something like :

file: `S00debug` 
```
#!/bin/sh

echo "start config"
ls /configs
echo "end config"
env
````
If it's blank between "start config" "end config" then the issue is probably that the `/configs` directory was clobbered.

if new log files, what do they say? how far do they get compared to the previous successful log?
